src/content/eth2/docking/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: The docking
+description: Learn about the docking - the first major eth2 upgrade to Ethereum.
+lang: en
+template: eth2
+sidebar: true
+---
+
+# What’s different about the docking?
+
+The docking’s role will change over time but it’s a foundational component for the secure, sustainable and scalable Ethereum we’re aiming towards.
+
+The docking isn’t like the Ethereum mainnet of today. It doesn’t have accounts and it can’t handle smart contracts. It’s role is to conduct or coordinate the network.
+
+## What’s in the upgrade?
+
+### Introducing staking
+
+The docking will introduce the concept of staking to Ethereum. This will allow you to put your ETH towards securing the network. Think of this like a public good that will make Ethereum healthier and earn you more ETH in the process. In practice, it will involve you staking ETH in order to run software that keeps Ethereum going. With this software you’ll be processing transactions and creating new blocks in the chain.
+
+This is a lot easier to do than how the network is currently secured: mining.
+
+This extra accessibility will help make Ethereum more secure in the long run. The more people that participate in the network, the more decentralized and secure it will become.
+
+This is also an important change for the second ETH2 upgade: shard chains.
+
+### Setting up for shard chains
+
+Shard chains will be the second Eth2 upgrade. They’ll increase the capacity of network and improve transaction speed by extending the network to 64 blockchains. The docking is an important first step in introducing shard chains, because they require staking to work securely.
+
+Eventually the docking will also be responsible for randomly assigning stakers to validate shard chains. This is key to ensuring stakers can’t become bad guys and collude to take over a shard. Well, it means they have less than a 1 in a trillion chance. This is where the docking’s coordination role comes into play.
```

Comment:
I'd remove 'bad guys' language as protocol incentives are ethically neutral. So maybe something like "This is key to making it difficult for stakers to collude and take over a shard."